.env.example

```
@@ -0,0 +1 @@
+  VITE_API_GENERATIVE_LANGUAGE_CLIENT=your_gemini_api_key_here
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)